src/test/softserve/academy/edu1/hw5/taskHW3/CarTest.java

```diff
@@ -0,0 +1,45 @@
+package softserve.academy.edu1.hw5.taskHW3;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+class CarTest {
+
+    public static Car[] fillArray() {
+        int count = 4;
+        Car[] cars = new Car[count];
+        cars[0] = new Car("Sedan", 2013, 1.4);
+        cars[1] = new Car("Coupe", 2011, 1.8);
+        cars[2] = new Car("Crossover", 2021, 2.2);
+        cars[3] = new Car("Hatchback", 2011, 2.6);
+        return cars;
+    }
+
+    @Test
+    void testMaxYear() {
```

Comment:
If we make the same action before each test we must use @BeforeEach annotation 
private Car[] cars;

`@BeforeEach`
public void fillCars(){
    int carCount = 4;
    cars = new Car[carCount];
           cars[0] = new Car("Sedan", 2013, 1.4);
        cars[1] = new Car("Coupe", 2011, 1.8);
        cars[2] = new Car("Crossover", 2021, 2.2);
        cars[3] = new Car("Hatchback", 2011, 2.6);

}